### ISSUE TITLE
RUVs batch correction question

### ISSUE THREAD

**Issue by gianfilippo** [drisso/RUVSeq#7]
Hi,

I am using RUVs to try  and correct a perturbation experiment.

In this experiment a large number of knock-down  experiment (2 technical replicates) were done on a cell line. Each library was prepared in groups of 8 (4 pairs of technical replicates). Controls were unfortunately included only in some of these libraries. Overall there are 3 sequencing batches. I am aware that this is not an ideal experimental design.

I am trying to exploit the controls present in some libraries, in each sequencing batch, to see if I can correct for lib prep, seq batch, etc

I use the estimated hidden factors in a glm model in edgeR to test each perturbation (2 replicates) against all the controls.

svobjRUVs = RUVs(counts, cIdx=rep(T,nrow(counts)), k=K, scIdx=CTRL_IND, round=T)
 where I iterate over K. CTRL_IND is the column index of the control samples in counts

I then do a PCA on the svobjRUVs$normalizedCounts and plot the resulting first and second principal components.

As I iterate over K I see the control samples converging, say, with k going from 1 to 5

What I notice, though, is that the other samples do not seem to be 'changing' as much change as the controls. It seems like the estimated hidden factors are affecting the control samples more than the others. Defeating the purpose of estimating confounders from samples that should not be affected by biological variation.

Any suggestions ?

Thanks

**Comment by drisso** [comment]
Hi @gianfilippo,

first of all, thanks for your interest in RUVSeq. This is an interesting question, but it's not related to issues with the software, rather with its use. Hence, the appropriate channel for this question is [the Bioconductor support forum](https://support.bioconductor.org/), in which other experts can weigh in with their thoughts.

I would ask that you post your question there, with a few additional details, e.g., how CTRL_IND actually looks like and perhaps with the PCA plots before and after RUV correction.

It is rather difficult to answer this question without those details, and even with them it might be that the controls don't fully capture the technical variation of the non-control samples. Again, it's difficult to predict whether that's the case without having more details on the experimental design and what actually constitute a group and a control sample.

To sum up, my suggestion is to post this question on the support forum with the added details outlined above.

**Comment by gianfilippo** [comment]
thanks
